ansible/roles/node/handlers/main.yml

```diff
@@ -3,6 +3,7 @@
   command: systemctl --system daemon-reload
   notify:
     - restart daemons
+  when: ansible_distribution != "Ubuntu"
```

Comment:
Nope. "init" isn't an setup fact/var. I made the checks predicated to ubuntu less-than 15.x.
14.04 is not going to get systemd. 16.04 should default to systemd but that is still at least 6 months away.